lib/model/message_list.dart

```diff
@@ -105,6 +105,8 @@ enum FetchingStatus {
 ///
 /// This comprises much of the guts of [MessageListView].
 mixin _MessageSequence {
+  bool get oneMessagePerBlock;
```

Comment:
Conceptually I think this should be private. It is convenient to use it in checkInvariants; but then let's mark it `@visibleForTesting`.